portfolio/src/main/webapp/index.html

```diff
@@ -89,13 +88,18 @@ <h2>Add any pre-scheduled events here</h2>
           <input type="time" id="new-event-end-time" name="new-event-end-time" value="20:00" />
         </span>
         <br />
-        <button type="button" class="btn btn-primary" onclick="createNewCalendarEvent()">Add this task</button>
+        <button type="button" class="btn btn-primary" onclick="createNewCalendarEventUserInput()">Add this task</button>
       </div>
     </div>
 
     <div id="new-event-list-wrapper">
       <h2>Events added:</h2>
-      <h3 id="event-end-time-warning">End time must be later than start time.</h3>
+      <div class="alert alert-secondary" role="alert" id="event-end-time-warning">
```

Comment:
Usually you'd have one alert element, and you can use Javascript to control the warning message. I'd recommend that. For example, initially the alert element can have the d-none (bootstrap) class name to be hidden, and it can be empty. And when you need to show it you can use javascript to set the inner text and remove the d none class.

Reply:
Updated. Now there is only one warning text box.